content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/source/touchui-limit-parsys.js

```diff
@@ -23,6 +23,7 @@
  * eg. to limit the components to 4 on rightpar of /content/geometrixx/en.html
  * set acsComponentsLimit=4 on /etc/designs/geometrixx/jcr:content/homepage/rightpar
  */
+ window.Granite.author = window.Granite.author || {};
```

Comment:
@schoudry there are no guarantees that Granite.author will be available before the extension anyway, since it is not a dependency of the clientlib. This fix will at least pass the reference to Granite.authoring. By the time the functions in the extension are called, Granite.author would have been loaded. again, no guarantees there, but I'd assume the extension wont be usable untill the page has loaded.

so, there are tow solutions here,
1. wait for document ready, then have the extension available. or
2. move this file into it's own clientlib and make sure that it's loaded after Granite.author, through some dependency